articles/tokens/access-token.md

```diff
@@ -21,6 +21,10 @@ The [Auth0 Management API v1](/api/management/v1) (which has been deprecated) us
 
 Access tokens are issued via Auth0's OAuth 2.0 endpoints: [/authorize](/api/authentication#authorize-client) and [/oauth/token](/api/authentication#get-token). You can use any OAuth 2.0-compatible library to obtain access tokens. If you do not already have a preferred OAuth 2.0 library, Auth0 provides libraries for many languages and frameworks that work seamlessly with our endpoints.
 
+:::note
+See the [Authentication API Explorer](/api/authentication#get-token) for additional usage notes on obtaining access tokens. 
+:::
+
 * Calls to the Lock widget will return an `access_token` as shown in the [Lock documentation](/libraries/lock).
 * [Examples using auth0.js](https://github.com/auth0/auth0.js).
 * Check the [List of tutorials](/tutorials) to see how to make calls to libraries for other languages/SDKs.
```

Comment:
You could also add a note after these bullets pointing to https://auth0.com/docs/libraries for more widgets and SDKs.